postreise/analyze/generation/curtailment.py

```diff
@@ -128,17 +129,17 @@ def calculate_curtailment_percentage_by_resources(scenario, resources=None):
 def calculate_curtailment_time_series_by_areas_and_resources(
     scenario, areas=None, resources=None
 ):
-    """Calculate a time series of curtailment for a set of valid areas and resources.
+    """Calculate hourly curtailment of each generator located in area(s) area and
+    fueled by resource(s)
 
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance.
     :param dict areas: keys are area types ('*loadzone*', '*state*' or
-        '*interconnect*'), values are a list of areas. Default is the interconnect of
-        the scenario. Default is the scenario interconnect.
+        '*interconnect*'), values are a list of areas. Default is the scenario
+        interconnect.
```

Comment:
`interconnect` -> `interconnect(s)`, since the scenario may be `"USA"`, `["Eastern", "Western"]`, etc.

Reply:
Done